_inc/lib/core-api/class.jetpack-core-api-module-endpoints.php

```diff
@@ -1066,6 +1070,126 @@ private function _process_onboarding( $data ) {
 			: join( ', ', $error );
 	}
 
+	static function get_first_sidebar() {
+		$active_sidebars = get_option( 'sidebars_widgets', array() );
+		unset( $active_sidebars[ 'wp_inactive_widgets' ], $active_sidebars[ 'array_version' ] );
+
+		if ( empty( $active_sidebars ) ) {
+			return false;
+		}
+		$active_sidebars_keys = array_keys( $active_sidebars );
+		return array_shift( $active_sidebars_keys );
+	}
+
+	static function add_business_address( $data ) {
+		$first_sidebar = self::get_first_sidebar();
+
+		if ( $first_sidebar ) {
+			$title = $data['businessAddress'][ 'name' ];
+			$address =
+					$data[ 'businessAddress' ][ 'city' ] . ' ' .
```

Comment:
Indentation appears a little off here. Do we want to make this one a little smarter, by grabbing all values, then filtering out any empty ones, and then building the string?

Reply:
Where are you prettify? 😆